Helper/PaymentResponseHandler.php

```diff
@@ -247,11 +250,19 @@ public function handlePaymentResponse($paymentsResponse, $payment, $order = null
                         }
                     } else {
```

Comment:
What do you think for adding a `elseif` block instead of just `else` with the condition of `$paymentInstanceCode === AdyenCcConfigProvider::CODE` and remove this condition from line 263?

Reply:
I like it, way cleaner